trie/database.go

```diff
@@ -701,6 +707,23 @@ func (db *Database) Commit(node common.Hash, report bool, callback func(common.H
 	start := time.Now()
 	batch := db.diskdb.NewBatch()
 
+	db.lock.Lock()
+	for _, v := range db.rawDirties {
+		batch.Put(v.K, v.V)
+	}
+	for k := range db.rawDirties {
```

Comment:
Use `make(KvMap)` be more simplely.

Reply:
not completely clear where should we use `make(KvMap)` here. Do you mean the `db.rawDirties`?